thesis-156/run-thesis-156.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+docker network create -d bridge --subnet=172.20.0.0/16 --gateway=172.20.0.1 thesis-156
+
+docker build --file THESIS-155/Dockerfile -t thesis-155:latest .
+
+docker run -d --network=thesis-156 --ip=172.20.0.5 --name=db-server thesis-155:latest
+
+docker run -d --network=thesis-156 --ip=172.20.0.10 --name=db-admin --link postgres:db -p 10081:8080 adminer:4.7.5-standalone
```

Comment:
@TanooKacha 
- What is this `--link postgres:db` for?
- ทำไมถึงเลือกใช้ `adminer:4.7.5-standalone`
- ปกติเราไม่ค่อยเขียนสคริปหรือโค้ดบรรทัดเดียวยาวๆ เป็นกิโลกกันแบบนี้ ตัดบรรทัดเถอะครับ จะได้ไม่เป็นภาระแก่ลูกหลาน
```suggestion
docker run -d --network=thesis-156 --ip=172.20.0.10 \
     --name=db-admin --link postgres:db -p 10081:8080 \
     adminer:4.7.5-standalone
```

Reply:
1 ยังไม่เข้าใจเรื่องของการ  link  เท่าที่ควร 
2เลือกใช้ตัวนี้เพราะเป็นตัวพื้นฐานที่แนะนำเนื่องจากทดลองใช้จึงเลือกใช้ตัวนี้
3 จะเอาไปปรับปรุงในครั้งต่อๆไป

Reply:
- เรื่อง link ก็ตามที่บอกว่าน่าจะเป็นอย่างนั้น